src/ansys/mapdl/core/misc.py

```diff
@@ -423,47 +424,76 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def load_file(mapdl, fname):
+def load_file(mapdl, fname, priority_mapdl_file=None):
     """
     Provide a file to the MAPDL instance.
 
-    If in local:
-        Checks if the file exists, if not, it raises a ``FileNotFound`` exception
+    Parameters
+    ----------
+    mapdl
+      Mapdl instance
+
+    fname : str, path
+      Path to the file.
+
+    priority_mapdl_file : bool
+      In case of the file existing in the MAPDL file and
+      in the Python local environment, this kwarg specifies
+      which one has priority.
+      By default is True.
```

Comment:
```suggestion
      In case of the file existing in the MAPDL environment and
      in the local Python environment, this parameter specifies
      which one has priority. Defaults to ``True``, meaning the MAPDL file
      has priority.
```